app/src/lib/git/cherry-pick.ts

```diff
@@ -275,14 +277,15 @@ export async function getCherryPickSnapshot(
   }
 
   const count = commits.length - remainingShas.length
+  const commitSummaryIndex = (count > 0 ? count : 1) - 1
```

Comment:
Kind of a nit, but I find this easier to read (since my brain is already wired for 0-index 🤣 )
```suggestion
  const commitSummaryIndex = count > 0 ? count - 1 : 0
```

Feel free go ignore this, though!

Reply:
lol .. now that I come back to it.. that is harder to read.. it was a result of doing A then modifying to B without rethinking it.